bananas_api/new_upload/readers/newgrf.py

```diff
@@ -117,6 +117,7 @@ class Feature(enum.IntEnum):
     AIRPORT_TILES = 0x11
     ROADTYPES = 0x12
     TRAMTYPES = 0x13
+    ROAD_STOPS = 0x14
```

Comment:
The features < 0x100 are meant for the regular non-translated features.
I guess ROAD_STOPS should be among the 10x magic.

Reply:
Went for the 40x series, as it isn't a fake feature. Just a dynamic patch-pack only one :P

Reply:
With OpenTTD#10144 merged, I guess this will be 0x14 after all. Changed back my PR to reflect that. Means GRFs build against upstream should also be detected correctly.